### ISSUE TITLE
on line team out of date

### ISSUE THREAD

**Issue by rmalina** [cassininazir/creativedisturbance#110]
the team on teh web site
http://creativedisturbance.org/meet-the-team/
is out of date

stephen witt should be on the alumnae page
and the new team members need to be added

roger

**Comment by vivraman** [comment]
Roger,

we're still waiting on the new members to send me their profiles

Vivek

**Comment by cassininazir** [comment]
Duplicate of #104. @vivraman will complete this today.
